apps/govquests-frontend/tailwind.config.ts

```diff
@@ -5,9 +5,10 @@ const config: Config = {
   theme: {
     extend: {
       colors: {
-        primary: "#EBE5E3",
-        optimism: "#FF0421",
-        optimismForeground: "#FFFFFF",
+        primary: "white",
+        secondary: "#EDEDED",
+        secondaryHover: "#6F6F70",
+        secondaryDisabled: "#C4C4C4",
```

Comment:
Minor, but would prefer to have tailwind colors in kebab case. I think the statement above would work:

"secondary-hover": "#6F6F70",
"secondary-disabled": "#C4C4C4",

Reply:
Makes a lot of sense, nice catch!